test/validation/api/random/random.c

```diff
@@ -218,10 +218,10 @@ static void random_test_frequency(odp_random_kind_t kind)
 
 	printf("\n\n");
 
-	for (int bits = 1; bits <= 16; bits++) {
+	for (int bits = 1; bits <= 8; bits++) {
 		const uint32_t cells = 1 << bits;
 		const uint64_t num = expected * cells;
-		uint64_t f[256 * 256] = { 0 };
+		uint64_t f[1 << 8] = { 0 };
```

Comment:
Please use normal number presentation for easier readability.

Reply:
The reason why I put it that way was to highlight that it's the same 8 as in the loop condition `bits <= 8`. But I'll change it.